### DISCUSSION TITLE
ESP-IDF Constant Error Popping Up - The terminal process "C:\WINDOWS\System32\cmd.exe" terminated with exit code: 9009

### DISCUSSION THREAD

**Discussion by HopWorks** [microsoft/vscode-discussions#1345] [Extension Development QnA] [answered]
Hi All,
I just updated VSCODE to the latest version, and then installed the ESP-IDF extension and all is working great. But I keep getting this notification... 
The terminal process "C:\WINDOWS\System32\cmd.exe" terminated with exit code: 9009.
And I do not know what it is related to. I did have some issues before, before I sorted all them out. Now I can write and program my 
 ESP32S3 development boards just fine, but this notification keeps popping up and I am worried what it relates to or means.

Thanks for your valuable time!!!!

**Comment by gjsjohnmurray** [comment, marked as answer]
Please report this to the extension developer. The "Report Issue" option from VS Code's Help menu should make this easy.

This discussion forum is for developers of extensions, not users.
